language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -28,6 +20,12 @@ pub enum ParseError<L, E> {
     User { error: E },
 }
 
+impl<L> From<Error> for ParseError<L, Error> {
```

Comment:
If we end up keeping this code around, we should eventually just get rid of this ParseError type (like I already did for move-lang). It's just a hold-over from lalrpop. It's fine like this for now, though. I'm still hoping we can find a way to stop maintaining two compilers....

Reply:
Yeah, as discussed in person it would probably be worth tackling this error stuff even if we don't do much else with the current parser.